tests/test_mritopng.py

```diff
@@ -127,4 +127,18 @@ def test_auto_contrast(self):
         ])
 
         if not np.array_equal(result.image, expected):
-            raise Exception("Expected:\n%s\n\nActual:\n%s\n"%(expected, result.image))
\ No newline at end of file
+            raise Exception("Expected:\n%s\n\nActual:\n%s\n"%(expected, result.image))
+
+    def test_folder_convert(self):
+        curr_path = os.path.dirname(os.path.realpath(__file__))
+        mri_dir_path = os.path.join(curr_path, 'data', 'samples')
+        png_dir_path = os.path.join(curr_path, 'data', 'png_dir')
+
```

Comment:
Would a simple
```
mri_dir_path = './data/samples'
png_dir_path = './data/png_dir'
```
work? Have tested that it works locally, not sure how the circle.ci is setup.

Reply:
@kshitij12345 Your snippet, which is simpler would have worked in `circleci` or any linux / mac host but potentially **fail** on a **Windows** machine OR any non-unixy OS. 

You want to write your Python code such that it works on all platforms and hence `os.path.join(..)` protects you from the specific nuances of specific operating systems.

Reply:
Oh yeah. Makes sense.